compiler/src/dotty/tools/dotc/typer/ProtoTypes.scala

```diff
@@ -503,7 +503,8 @@ object ProtoTypes {
   }
 
   /**  Same as `constrained(tl, EmptyTree)`, but returns just the created type lambda */
```

Comment:
The documentation is now misleading, also is there a reason why `alwaysAddTypeVars = ctx.typerState.isCommittable` is not a good default for the other overload ?

Reply:
it would be a good default, except that the default value would refer to the `ctx` which follows.